js/sparkswap.js

```diff
@@ -271,7 +271,46 @@ module.exports = class sparkswap extends Exchange {
     }
 
     async fetchOrder (id, symbol = undefined, params = {}) {
-        return this.privateGetV1OrderId ({ id });
+        const order = await this.privateGetV1OrdersId ({ id });
+        let status = 'open';
+        if (order.status === 'CANCELLED') {
+            status = 'cancelled';
+        }
+        if (order.status === 'COMPLETED') {
+            status = 'closed';
+        }
+        let type = 'limit';
+        if (order.is_market_order) {
```

Comment:
Same thing here, why set limit as the default.

Reply:
@martineehrlich this is a different case. We dont actually return the type `limit` or `market`, we simply have a boolean that is either `true` for a market order, or false.